### PR TITLE
User API endpoints

### PR DESCRIPTION
**Pull request by juliasaveliff**
- Delete endpoint
- Get-many endpoint (with pagination)
- Pages count endpoint

### CODE REVIEW

File: rackcity/views/user_views.py [hunk 1]
```diff
@@ -84,3 +61,143 @@ def netid_login(request):
...
+@permission_classes([IsAuthenticated])
+def usernames(request):
+    """
+    Get all existing usernames.
+    """
+    usernames = [obj.username for obj in User.objects.all()]
+    return JsonResponse(
+        {"usernames": usernames},
+        status=HTTPStatus.OK,
+    )
```
Comment (juliasaveliff) [inline rackcity/views/user_views.py]: this is not new code --  just moved it down in the file

File: rackcity/views/it_instance_views.py [hunk 2]
```diff
@@ -533,7 +532,10 @@ def instance_page_count(request):
     Return total number of pages according to page size, which must be
     specified as query parameter.
     """
-    if not request.query_params.get('page_size') or int(request.query_params.get('page_size')) <= 0:
+    if (
+        not request.query_params.get('page_size')
+        or int(request.query_params.get('page_size')) <= 0
+    ):
```
Comment (juliasaveliff) [inline rackcity/views/it_instance_views.py]: no code changes just line length linting ;    )

Comment (benauriemma) [reply rackcity/views/it_instance_views.py]: :     )

Comment (benauriemma) [reply rackcity/views/it_instance_views.py]: I tried to make that smiley face longer but it wouldn't let me

Comment (juliasaveliff) [reply rackcity/views/it_instance_views.py]: :         )

Comment (juliasaveliff) [reply rackcity/views/it_instance_views.py]: same

**Pull request merged**